bin/run_pypit

```diff
@@ -57,10 +58,11 @@ def main() :
 
     # Load options from command line
     try:
-        opt, arg = getopt.getopt(sys.argv[1:], 'hmqc:v:', ['help',
+        opt, arg = getopt.getopt(sys.argv[1:], 'hmqcd:v:', ['help',
                                                            'use_masters',
                                                           'quick',
                                                           'cpus',
+                                                            'develop',
```

Comment:
Done.  I should note that there is something I don't understand about the parsing.
Am raising an issue..